pkg/cmd/version/version.go

```diff
@@ -0,0 +1,28 @@
+package version
+
+import (
+	"fmt"
+
+	"github.com/spf13/cobra"
+)
+
+// NewCmdVersion returns the current version of cli being used
+func NewCmdVersion(version, date string) *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Get the version of the DeepSource CLI",
+		Run: func(cmd *cobra.Command, args []string) {
+			fmt.Println(formatVersionOutput(version, date))
+		},
+	}
+	return cmd
+}
+
+func formatVersionOutput(version, date string) string {
+
+	// Changelog URL
+	releasePath := fmt.Sprintf("https://github.com/deepsourcelabs/cli/releases/tag/%s", version)
+
+	return fmt.Sprintf("DeepSource CLI %s\nReleased on %s\n%s", version, date, releasePath)
```

Comment:
We don't need the changelog URL. Too much information. Let the behaviour be like this:

```
$ deepsource version
DeepSource CLI version 1.5.0 (2021-01-21)
```

In brackets is the release date, of course.

Reply:
Ack. Making the changes